src/features/journey/components/Results/JourneyDetails.tsx

```
@@ -8,21 +13,44 @@ import {
   Share2,
   Footprints,
   Star,
+  Info,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)